.eslintrc.json

```diff
@@ -15,6 +15,7 @@
   "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],
   "plugins": ["react"],
   "rules": {
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": false, "optionalDependencies": false, "peerDependencies": false}],
```

Comment:
- [ ] Kindly ensure you do not add any configurations to the provided linter files.